com.github.Nheko.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
Needs `"rename-appdata-file": "nheko.appdata.xml"` and `"rename-desktop-file": "nheko.desktop"`, and `"rename-icon": "nheko"`.